src/topology/controller.rs

```diff
@@ -93,6 +98,41 @@ impl TopologyController {
             }
         }
 
+        /*
+        Without this block
+        the first config loaded determines whether the api is
+        always on or always off
+        */
```

Comment:
I'd prefer to see this phrased in the positive (i.e. what it does rather than what happens if it's not there) and it formatted in the same style as other comments.

Reply:
Sure thing. Let me know if the changed version matches your expectations.